library/std/src/os/wasi/mod.rs

```diff
@@ -1,3 +1,4 @@
+#![deny(unsafe_op_in_unsafe_fn)]
```

Comment:
Since this is the parent module, you shouldn't need it in all of the child modules.

Could you upgrade this to `forbid` to match the others?